template-for-repository/proofs/Makefile.common

```diff
@@ -339,63 +339,63 @@ goto: $(HARNESS_GOTO).goto
 ################################################################
 # Targets to run the analysis commands
 
-logs/result.txt: $(HARNESS_GOTO).goto
+$(LOGDIR)/result.txt: $(HARNESS_GOTO).goto
 	$(call DO_CBMC,$(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --trace,$<,$@)
 
-logs/result.xml: $(HARNESS_GOTO).goto
+$(LOGDIR)/result.xml: $(HARNESS_GOTO).goto
 	$(call DO_CBMC,$(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --trace --xml-ui,$<,$@)
 
-logs/property.xml: $(HARNESS_GOTO).goto
+$(LOGDIR)/property.xml: $(HARNESS_GOTO).goto
 	$(call DO_CBMC,$(CBMCFLAGS) $(CHECKFLAGS) --unwinding-assertions --show-properties --xml-ui,$<,$@)
 
-logs/coverage.xml: $(HARNESS_GOTO).goto
+$(LOGDIR)/coverage.xml: $(HARNESS_GOTO).goto
 	$(call DO_CBMC,$(CBMCFLAGS) $(COVERFLAGS) --cover location --xml-ui,$<,$@)
 
-result: logs/result.txt
+result: $(LOGDIR)/result.txt
 
-property: logs/property.xml
+property: $(LOGDIR)/property.xml
 
-coverage: logs/coverage.xml
+coverage: $(LOGDIR)/coverage.xml
 
-report: logs/result.txt logs/property.xml logs/coverage.xml
+report: $(LOGDIR)/result.txt $(LOGDIR)/property.xml $(LOGDIR)/coverage.xml
 	$(VIEWER) \
-	--result logs/result.txt \
-	--block logs/coverage.xml \
-	--property logs/property.xml \
+	--result $(LOGDIR)/result.txt \
+	--block $(LOGDIR)/coverage.xml \
+	--property $(LOGDIR)/property.xml \
 	--srcdir $(SRCDIR) \
 	--goto $(HARNESS_GOTO).goto \
```

Comment:
Don't you need here `$(GOTODIR)/$(HARNESS_GOTO).goto`?

Reply:
No. Line 229, `HARNESS_GOTO ?= $(GOTODIR)/$(HARNESS_FILE)`